test/e2e/config/azure-dev.yaml

```diff
@@ -153,13 +153,13 @@ providers:
 variables:
   AKS_KUBERNETES_VERSION: "latest"
   AKS_KUBERNETES_VERSION_UPGRADE_FROM: "latest-1"
-  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.24}"
-  FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.24}"
+  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-stable-1.25}"
+  FLATCAR_KUBERNETES_VERSION: "${FLATCAR_KUBERNETES_VERSION:-stable-1.25}"
```

Comment:
@invidian is there any reason not to update the default k8s version to test our flatcar reference cluster scenario against?

Reply:
Should be fine to update. I'll be making sure that at all new minor versions are available. It seems version discovery works as expected, as 1.25.4 was already available. I'll build 1.25.6 with new stable release now.